services/ui-src/src/components/pages/Export/ExportedReportPage.tsx

```diff
@@ -69,7 +74,62 @@ export const ExportedReportPage = () => {
 };
 
 export const reportTitle = (report: Report) => {
-  return `${report.state} ${report.name}`;
+  return `${StateNames[report.state]} ${getReportName(report.type)} for: ${
+    report.name
+  }`;
+};
+
+export const reportDetails = (report: Report) => {
+  return (
+    <Table variant={"reportDetails"}>
+      <Thead>
+        <Tr>
+          <Th>Reporting year</Th>
+          <Th>Last edited</Th>
+          <Th>Edited by</Th>
+          <Th>Status</Th>
+        </Tr>
+      </Thead>
+      <Tbody>
+        <Tr>
+          <Td>{report.year}</Td>
+          <Td>{formatMonthDayYear(report.lastEdited!)}</Td>
+          <Td>{report.lastEditedBy}</Td>
+          <Td>{report.status}</Td>
+        </Tr>
+      </Tbody>
+    </Table>
+  );
+};
+
+export const reportSubmissionSetUp = (report: Report) => {
+  const rows = [
+    {
+      indicator: "Is your state reporting on the HCBS CAHPS Survey?",
+      response: report.options.cahps ? "Yes" : "No",
+    },
+    {
+      indicator: "Is your state reporting on the NCI-IDD Survey?",
+      response: report.options.nciidd ? "Yes" : "No",
+    },
+    {
+      indicator: "Is your state reporting on the NCI-AD Survey?",
+      response: report.options.nciad ? "Yes" : "No",
+    },
+    {
+      indicator: "Is your state reporting on the POM Survey?",
+      response: report.options.pom ? "Yes" : "No",
+    },
+  ];
```

Comment:
Correct me if Im wrong, but I believe these submission questions only apply for the QMS report, so we dont need to render this for the other reports.